doc/Actions.md

```diff
@@ -0,0 +1,75 @@
+# Provided actions
+
```

Comment:
Add: "Actions in Kitodo.Mediaserver can be anything that can be performed on works or work files: conversions, creation of derivatives, manipulations of the mets file, setting of access rights etc. Actions may be executed by import, from the admin ui or using the cli tool. For examples of action configurations, see the [configuration file documentation](Configuration-file.md)"